cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -1077,6 +1077,9 @@ rules:
   - apiGroups: [""]
     resources: ["persistentvolumeclaims/status"]
     verbs: ["update", "patch"]
+  - apiGroups: [""]
```

Comment:
This change is also needed in the helm chart and in ceph_manifests.go for the tests.

Reply:
missed to add that. updated now.